py/tests/integration/data/bchain.txt

```diff
@@ -1 +1 @@
\ No newline at end of file
+[{"nonce": 0, "target": 553713663, "transactions": null, "pow": null, "time": 0, "height": 0, "version": 1, "txs_hash": null, "prev_hash": null, "state_hash": null}]
```

Comment:
Not sure this is valid. How did you generate it? Cc @velzevur

Reply:
Akos' work I'm afraid...

Reply:
Please ignore `py/tests/integration/data/bchain.txt`
I'll have it refactored in a subsequent PR

Reply:
Os this this bchain.txt change shall be removed from this PR.